contents/blog/cookieless-analytics.md

```diff
@@ -0,0 +1,41 @@
+---
+date: 2021-08-13
+title: Cookieless Analytics - How To Track Website and App Users Without Third-Party Cookies
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: marcus-hyett
+featuredImage: ../images/blog/filename.png
+featuredImageType: full
+---
+
+Google has stated that they intend to deprecate the use of 3rd party cookies in 2023. But why is this important, and how will this affect product analytics?
```

Comment:
What does this specifically mean for someone less informed (like me) - Google will automatically block all 3rd party cookies in Chrome? Or does Google have some sort of power to block 3rd party cookies across the internet generally?

Reply:
Just In chrome, but its basically all of the internet since Chrome accounts for 70% and Safari has already done it... so 80% will be blocked by then

Reply:
I think it's definitely worth putting in something along those lines - the way this is written is quite vague at the moment, and your explanation makes that a whole lot clearer!